src/types/Cart.ts

```diff
@@ -0,0 +1,108 @@
+import { Image } from './Image';
+import { PageInfo } from './PageInfo';
+import { Product, Variant } from './Product';
+
+export enum DiscountType {
+  FIXED_AMOUNT,
+  FIXED_PERCENTAGE,
+}
+
+export type Discount = {
+  readonly amount: number;
+  readonly amountV2: number;
+  readonly description: string;
+  readonly title: string;
+  readonly value: number;
+  readonly valueType: DiscountType;
+};
+
+export enum CurrencyCode {
+  INR,
+}
+
+export type TaxLine = {
+  readonly rate: number;
+  readonly ratePercentage: number;
+  readonly title: string;
+};
+
+export type CartLineItem = {
+  readonly appliedDiscount: Discount;
+  readonly discountedTotal: string;
+  readonly discountedUnitPirce: string;
+  readonly id: string;
+  readonly image: Image;
+  readonly name: string;
+  readonly originalTotal: string;
+  readonly originalUnitPrice: string;
+  readonly product: Product;
+  readonly quantity: number;
+  readonly sku: string;
+  readonly taxLines: readonly TaxLine[];
+  readonly texable: boolean;
```

Comment:
Typo. Please do two things:
1. Install any spell checker extension
2. Copy paste properties, always.

Please check all of these. I haven't checked all